1237-registrator_worklist/creator_adds_a_new_message.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Creator adds a new message
+
+    @1249
+    Scenario: Creator adds a new message on a DOI request
```

Comment:
Er det ikke mer `Creator sends a Message to a Curator`? synes det er så rart å si `on a DOI request`

Reply:
høres litt rart ut ja, men det er forskjell på å legge til en kommentar på DOI request (dette scenario) og Support request (scenario under), så det må jo komme frem på en måte. mottaker kan være hvilken som helst Curator også, i tillegg til at man kan se sin egen melding selv etterpå, så det er mer en del av DOI-objektet enn en direkte melding til en curator